### DISCUSSION TITLE
[灌水]以太适性自测结果是这个分数，我能去空洞里搬砖吗?

### DISCUSSION THREAD

**Discussion by waerte-yang** [share121/inter-knot#3087] [其他]
![IMG_20240819_170518](https://github.com/user-attachments/assets/05ee44f2-89c6-4eb8-b1fb-82bef3cf6df9)
本人学生，高中还没毕业。老爸因为工作原因买了以太适性自测卡，我出于好奇就顺便也跟着测了一下，结果拿了个80分。老爸当时就从椅子上跳起来，逼着我重新测了一次·据说50分是能从事空洞相关工作的及格线，80分是很高的分数了，这是不是意味着我也可以去空洞里当搬运工了?

**Comment by lllliufeng** [comment]
有这分数还**~~要啥自行车~~**当啥搬运工，**王牌调查员**的位置就是你的啊！

**Comment by yivitte** [comment]
怀疑楼主是在凡尔赛，却没找到证据

**Comment by wuyueji** [comment]
建议楼主跑去空洞里看见遗骸就是TM吃吃吃吃吃，死路屠夫吃吃吃，恶名杜拉罕吃吃吃，空洞治退就靠你了

**Comment by Aha-SWYH** [comment]
小时候为了以后能从事空洞工作，熬了半个月恶补空洞注意事项。结果得了个40分，我们要不换一下吧

**Comment by GrunderIndustries** [comment]
创立者工业诚邀您成为我司的机密设备测试员

**Reply by Fengnix216** [reply]
我83分能当吗？当了的话可以不用再回墨提斯听那帮老毕等叨叨了吗？

**Comment by Lyou777** [comment]
楼主我现在在外面，您可以帮我测一下吗？ 





整活: )<img src="https://bbs-static.miyoushe.com/static/2023/11/16/44d02375dbff4fde5ea3eadc82730131_8648553945620661617.png" alt="厨师邦布-疲惫" width="75" >

**Comment by trifortune** [comment]
也。。。从楼主这么高以太适性以及你对父亲的描述来看，你家的工作真没那么简单吧。。。<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by sr-BEEP** [comment]
什么感觉在哪里见过这帖子........

**Comment by chihung3927** [comment]
这辈子有了!

**Comment by ALXY-12** [comment]
这位同学，你好，墨提斯院诚邀你的加入!墨提斯院建校历史悠久，师资雄厚，以培养高端研究人才为己任。若确认你的以太适性分数无误，我们热忱邀请你加入以太适性班，无门槛办理转学手续，学费减半并设有丰厚奖学金。在那里你一定能踏上理想的道路，发挥你非同寻常的才能。详情已发送私信，若有兴趣请查收和回复。<img src="https://bbs-static.miyoushe.com/static/2024/01/31/42b5ef8bab5a58fbdd74f17c21c12420_2665774360840406961.png" alt="花火 略略略" width="75" >

**Reply by Fengnix216** [reply]
83以太适性已经被拐进去了，别来！太噩梦了😭

**Comment by kira-1-kira** [comment]
我捡到一个类似的，不知道是不是以太适性测试的，我测了一下上面显示2条杆是怎么回事<img src="https://img-static.mihoyo.com/communityweb/upload/53d67f3907a2c715abf86bf07f3ceefb.png" alt="放大镜" width="75" >

**Reply by Guessme404** [reply]
羊了个羊

**Comment by Leonie0202** [comment]
啊，好羡慕啊，像我们这种以太适性低的就不要考虑去空洞内工作了...

**Comment by Fengnix216** [comment]
没关系bro，至少你不用在墨提斯听那帮老毕等叨叨，我听了快1年感觉耳朵里面要生各种恶名了
________________________________
发件人: Leonie0202 ***@***.***>
发送时间: 2024年8月21日 23:56
收件人: share121/inter-knot ***@***.***>
抄送: Fengnix216 ***@***.***>; Comment ***@***.***>
主题: Re: [share121/inter-knot] [灌水]以太适性自测结果是这个分数，我能去空洞里搬砖吗? (Discussion #3087)


啊，好羡慕啊，像我们这种以太适性低的就不要考虑去空洞内工作了...

―

**Reply by Leonie0202** [reply]
欸..看样子真的好惨的样子，所以我该庆幸吗

**Comment by fujinhui** [comment]
这个委托我接了！
